.github/composite-actions/major-version-upgrade-util/action.yml

```diff
@@ -0,0 +1,82 @@
+name: 'Major Version Upgrade Utility'
+inputs:
+  engine_branch: 
+    description: "Engine Branch"
+    required: true
+  extension_branch: 
+    description: "Extension Branch"
+    required: true
+  is_final_ver:
+    description: "Is this the final version"
+    required: true
+  prev_install_dir: 
+    description: "Previous version installed in this directory"
+    required: false
+    default: postgres13
```

Comment:
why is this postgres13?

Reply:
made it a required input parameter, so removed default field